services/galley/schema/src/Main.hs

```diff
@@ -98,6 +98,9 @@ main = do
       V52_FeatureConferenceCalling.migration
       -- When adding migrations here, don't forget to update
       -- 'schemaVersion' in Galley.Data
+      --
+      -- FUTUREWORK: once #1726 has made its way to master/production,
+      -- the 'message' field in connections table can be dropped.
```

Comment:
Why does this go as a separate PR, and not part of this PR?

Reply:
I wrote some documentation about this, here: https://github.com/wireapp/wire-server/pull/1747/files